pyphare/pyphare/pharesee/hierarchy.py

```diff
@@ -701,10 +705,10 @@ def plot1d(self, **kwargs):
                 if qty is None:
                     qty = list(patch.patch_datas.keys())[0]
 
-                layout  = patch.patch_data[qty].layout
-                nbrGhosts = layout.nbrGhostsFor(qty)
-                val = patch.patch_datas[qty].dataset[nbrGhosts:-nbrGhosts]
-                x   = patch.patch_datas[qty].x[nbrGhosts:-nbrGhosts]
+                layout  = patch.patch_datas[qty].layout
+                nbrGhosts = layout.nbrGhostFor(qty)
+                val = patch.patch_datas[qty].dataset[nbrGhosts[0]:-nbrGhosts[0]]
```

Comment:
if this is 1d . this could be 

```
val = patch.patch_datas[qty][patch.box]
```

pretty sure it handles primals as expected

Reply:
can we really throw a box at a h5 dataset? :/

Reply:
this is in plot1d so I think it's ok.
btw it's nice to be able to do pd[box] but we still have to handle .x[ ] manually

Reply:
> handle .x[ ] manually

yeah, you can do something like [this](https://github.com/PhilipDeegan/PHARE/blob/3d_base/tests/simulator/test_initialization.py#L307)

so it might be `patch.patch_datas[qty].x[select[0]]`